src/main/java/frc/robot/RobotContainer.java

```diff
@@ -113,16 +111,15 @@ private void configureButtonBindings() {
     buttons[Constants.rightBumperButton].whileHeld(intakeReverseCommand);
     
     // CDS
-    buttons[Constants.YButton].whileHeld(CDSForwardCommand);
-    buttons[Constants.BButton].whileHeld(CDSReverseCommand);
+    buttons[Constants.leftBumperButton].whileHeld(CDSForwardCommand);
+    buttons[Constants.rightBumperButton].whileHeld(CDSReverseCommand);
```

Comment:
in conflict with cycling aim mode down (rightBumperButton = downbutton in Constants)